src/encryption/path.go

```diff
@@ -34,26 +30,3 @@ func (v mapStep) String() string {
 func (v sliceStep) String() string {
 	return fmt.Sprintf("[%v]", int(v))
 }
-
-func (v sliceStep) Get(object interface{}) interface{} {
-	slice := object.([]interface{})
-	return slice[v]
-}
-
-func (v mapStep) Get(object interface{}) interface{} {
-	m := object.(orderedmap.OrderedMap)
-	result, _ := m.Get(v.String())
-	return result
-}
-
-func (v mapStep) Update(object interface{}, value interface{}) interface{} {
-	m := object.(orderedmap.OrderedMap)
-	m.Set(v.String(), value)
-	return m
-}
-
-func (v sliceStep) Update(object interface{}, value interface{}) interface{} {
-	slice := object.([]interface{})
-	slice[v] = value
-	return slice
-}
```

Comment:
stary nacrt implementacie update funkcie. Povodne som to mal cez metody step interface ale vypadalo to divne tak som to prepisal vid nizsie.